cmd/kion/setup/setup.go

```diff
@@ -194,11 +194,22 @@ func run() error {
 		return err
 	}
 
+	var govcloudRegion string
+	err = survey.AskOne(
```

Comment:
govcloud is such a niche feature that I'm not sure it makes sense to always ask about it in setup. what do you think @jonahb ?

Reply:
I agree but I don't think we need this setting (see review comment).